### PR TITLE
Update README for header validator

### PR DESCRIPTION
**Pull request by linnan-github**

### CODE REVIEW

File: header-validator/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Run tests
 
 `npm run test`
-`npm run test-source`
-`npm run test-trigger`
+`npm run test-eligible`
+`npm run test-json`
+`npm run test-os`
```
Comment (apasel422) [inline header-validator/README.md:6]: I think we just need a single command: `npm run test`.

https://github.com/WICG/attribution-reporting-api/blob/6393bc05de80f1132eda1ba244e1330deaa2cc4e/header-validator/package.json#L7

**Review by maudnals — APPROVED** [review]

File: header-validator/README.md (hunk 1 above)
Comment (linnan-github) [reply header-validator/README.md:6]: yeah, but probably it doesn't hurt to list all tests available?

Comment (apasel422) [reply header-validator/README.md:6]: I don't think it's beneficial to list them redundantly here; as evidenced by this PR itself, it's easy for the doc to become out of date with respect to the actual package.

Comment (linnan-github) [reply header-validator/README.md]: That's fair, updated with only `npm run test`, thanks.

**Pull request merged**